solo-chains/runtime/dancelight/src/lib.rs

```diff
@@ -2045,6 +2045,9 @@ impl pallet_registrar::Config for Runtime {
     type DepositAmount = DepositAmount;
     type RegistrarHooks = DancelightRegistrarHooks;
     type RuntimeHoldReason = RuntimeHoldReason;
+    #[cfg(feature = "runtime-benchmarks")]
+    type InnerRegistrar = ();
+    #[cfg(not(feature = "runtime-benchmarks"))]
     type InnerRegistrar =
         InnerDancelightRegistrar<Runtime, AccountId, Registrar, paras_registrar::TestWeightInfo>;
```

Comment:
Now that we are touching this, we should probably change `paras_registrar::TestWeightInfo` for weights::runtime_common_paras_registrar::SubstrateWeight<Runtime>;

Reply:
Updated in https://github.com/moondance-labs/tanssi/pull/783/commits/f0c48c3e3ddb550cac505da66391875803d9cfde